docs/mkdocs.yml

```diff
@@ -92,298 +91,229 @@ plugins:
-                - tutorials/multi_agent/agent_supervisor.ipynb
-                - tutorials/multi_agent/hierarchical_agent_teams.ipynb
-            - Planning Agents:
-                - tutorials/plan-and-execute/plan-and-execute.ipynb
-                - tutorials/rewoo/rewoo.ipynb
-                - tutorials/llm-compiler/LLMCompiler.ipynb
-            - Reflection & Critique:
-                - tutorials/reflection/reflection.ipynb
-                - tutorials/reflexion/reflexion.ipynb
-                - tutorials/tot/tot.ipynb
-                - tutorials/lats/lats.ipynb
-                - tutorials/self-discover/self-discover.ipynb
-        - Evaluation & Analysis:
-            - tutorials/chatbot-simulation-evaluation/agent-simulation-evaluation.ipynb
-            - tutorials/chatbot-simulation-evaluation/langsmith-agent-simulation-evaluation.ipynb
-        - Experimental:
-            - tutorials/storm/storm.ipynb
-            - tutorials/tnt-llm/tnt-llm.ipynb
-            - tutorials/web-navigation/web_voyager.ipynb
-            - tutorials/usaco/usaco.ipynb
-            - tutorials/extraction/retries.ipynb
-        - LangGraph Platform:
-            - tutorials/auth/getting_started.md
-            - tutorials/auth/resource_auth.md
-            - tutorials/auth/add_auth_server.md
-    - Resources:
-      - Companies using LangGraph: adopters.md
-      - LLMS-txt: llms-txt-overview.md
-      - FAQ: concepts/faq.md
-      - Troubleshooting:
-          - Troubleshooting: troubleshooting/errors/index.md
-          - troubleshooting/errors/index.md
-          - troubleshooting/errors/GRAPH_RECURSION_LIMIT.md
-          - troubleshooting/errors/INVALID_CONCURRENT_GRAPH_UPDATE.md
-          - troubleshooting/errors/INVALID_GRAPH_NODE_RETURN_VALUE.md
-          - troubleshooting/errors/MULTIPLE_SUBGRAPHS.md
-          - troubleshooting/errors/INVALID_CHAT_HISTORY.md
-          - troubleshooting/errors/INVALID_LICENSE.md
-          - troubleshooting/studio.md
-      - LangGraph Academy Course: https://academy.langchain.com/courses/intro-to-langgraph
-
-  - Agents:
-    - agents/overview.md
-    - Get started:
-      - agents/agents.md
-    - Documentation:
-      - agents/run_agents.md
-      - agents/streaming.md
-      - agents/models.md
-      - agents/tools.md
-      - agents/mcp.md
-      - agents/context.md
-      - agents/memory.md
-      - agents/human-in-the-loop.md
-      - agents/multi-agent.md
-      - agents/evals.md
-      - agents/deployment.md
-      - agents/ui.md
-    - Resources:
-      # NOTE: prebuilt.md is auto-generated by `make build-prebuilt`
-      - agents/prebuilt.md
+        - Local deploy: tutorials/langgraph-platform/local-server.md
+        - Template applications: concepts/template_applications.md  # TODO: make tutorial
+        - Cloud deploy: cloud/quick_start.md
+    - LangGraph:
+      # - how-tos/index.md
+      - concepts/functional_api.md
```

Comment:
not sure if it makes sense for functional API to be the first thing a user sees here

<img width="315" alt="image" src="https://github.com/user-attachments/assets/27dd610e-6c1a-473d-88ab-43a87615935b" />

also, personally, i would push the LangGraph Runtime doc to below - this is kind of advanced section IMO